Library/Homebrew/cmd/list.rb

```diff
@@ -140,6 +140,8 @@ def initialize(path)
           # dylibs have multiple symlinks and we don't care about them
           (pnn.extname == ".dylib" || pnn.extname == ".pc") && !pnn.symlink?
         end
+      when ".bottle"
+        # Ignore .bottle
```

Comment:
Do you mean `brew list hello`? Sorry if I misunderstand the question. The goal of this line is to elide `.bottle/hello.rb` from the output of `brew list hello`.